### PR TITLE
Strip all special characters

### PR DESCRIPTION
**Pull request by ancy2023**

### CODE REVIEW

**Comment by ancy2023** [comment]
@incraigulous I have committed the login file unknowingly here, which needs to be committed in the login button fix branches, please review and merge that also, sorry for the inconvenience

**Comment by incraigulous** [comment]
> @incraigulous I have committed the login file unknowingly here, which needs to be committed in the login button fix branches, please review and merge that also, sorry for the inconvenience

Understood! I will wait until the login button fix is complete and merged before I merge this in.

**Review by incraigulous — APPROVED** [review]

**Pull request merged**
